exercises/calculator.cpp

```diff
@@ -5,8 +5,30 @@
   Insert first number: 4
   Insert second number: 2
 
-  SUM: 6
+  Sum: 6
   Difference: 2
   Multiplication: 8
   Division: 2
 */
+
+#include <stdlib.h>
+#include <stdint.h>
+#include <stdio.h>
+
+float x, y;
+
+int main(int argc, char **argv){
+	printf("First number: ");
+	scanf("%f", &x);
+
+	printf("Second number: ");
+	scanf("%f", &y);
+
+	printf("\n");
+	printf("Sum: %.2f\n", x + y);
+	printf("Difference: %.2f\n", x - y);
+	printf("Multiplication: %.2f\n", x * y);
+	printf("Division: %.2f\n", x / y);
+
+	return 0;
```

Comment:
potresti sostituire 0 con EXIT_SUCCESS in modo da migliorare la leggibilità del codice
```c++
#define EXIT_SUCCESS 0
...
return EXIT_SUCCESS;
```